scripts/pre_commit_linter.py

```diff
@@ -116,22 +121,66 @@ def _get_changed_filenames():
     return unstaged_files + staged_files
 
 
-def _get_all_files_in_directory(dir_path):
+def _does_glob_pattern_include_filename(filename, glob_patterns):
```

Comment:
Ah, looks like this has been pluralized. Suggest changing to: "is_filename_in_glob_patterns"